src/styles/blocks/timeless.scss

```diff
@@ -0,0 +1,62 @@
+.timeless {
+  background-color: #fcfaef;
+  margin-bottom: 64px;
+
+  &__grid {
+    @include pageGrid;
+  }
+
+  @include onTablet {
+    margin-bottom: 120px;
+  }
+
+  &__title {
+    margin-bottom: 24px;
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-column: span 6;
+    }
+  }
+
+  &__image {
+    width: 100%;
+    height: 200px;
+
+    object-fit: cover;
+    object-position: center;
+    margin-bottom: 48px;
+
+    @include onTablet {
+      height: 400px;
+      margin-bottom: 120px;
+    }
+
+    @include onDesktop {
+      height: 460px;
+    }
+  }
+
+  &__text {
+    font-weight: 400;
```

Comment:
It is better to use variables for font-weight instead of directly using the numeric value. Consider using a variable like $font-weight-normal.